tests/test_plotfileparticledata.py

```diff
@@ -0,0 +1,50 @@
+from dataclasses import dataclass
+
+import amrex.space3d as amr
+
+if not amr.initialized():
+    amr.initialize([])
+
+plt_file_name = "plt0000600"
+
+
+@dataclass
+class Particle:
+    x: float
+    y: float
+    z: float
+
+    ib_id: int
+    idx: int
+
+
+def load_data(plot_file_name):
+    plt = amr.PlotFileData(plt_file_name)
+
+    probDomain = plt.probDomain(0)
```

Comment:
For pytest, you an start those with `def test_...` to be picked up.

if you like to run tests as standalone, you could add a main to the file, but usually we run them via pytest.